src/Airspace/NearestAirspace.hpp

```diff
@@ -35,13 +35,13 @@ class ProtectedAirspaceWarningManager;
 class NearestAirspace {
 
 public:
-  const AbstractAirspace *airspace;
+  const AbstractAirspace *airspace = nullptr;
 
   /**
    * The horizontal or vertical distance [m], depending on which
    * function filled this object.
    */
-  double distance;
+  double distance = 0.0;
```

Comment:
And again an initialiser with an arbitrary value. This pretends the distance to the nearest airspace is zero, when there is not actually any airspace. To fix the problem mentioned in the commit message, a different solution must be chosen that can be done without returning bogus values.

Reply:
Disable the warning in Minimum.hpp.
I re-used your text from similar occurrences.